Libplanet.Tests/Net/IceServerTest.cs

```diff
@@ -19,8 +19,8 @@ public async Task CreateTurnClient()
             var userInfo = turnUri.UserInfo.Split(':');
             var servers = new List<IceServer>()
             {
-                new IceServer(new[] { "turn:turn.does-not-exists.org" }),
-                new IceServer(new[] { "stun:stun.l.google.com:19302" }),
+                new IceServer(new[] { "turn://turn.does-not-exists.org" }),
+                new IceServer(new[] { "stun://stun.l.google.com:19302" }),
```

Comment:
How about making the `IceServer(string[])` constructor to throw an `ArgumentException` when the given URLs are unacceptable?